algolia-config.json

```diff
@@ -1,24 +1,28 @@
 {
   "index_name": "mina",
-  "start_urls": ["https://docs.minaprotocol.com"],
+  "start_urls": [
+    {
+      "url": "https://docs.minaprotocol.com"
+    }
+  ],
   "sitemap_urls": ["https://docs.minaprotocol.com/sitemap.xml"],
   "sitemap_alternate_links": true,
   "js_render": true,
-  "stop_urls": ["/tests"],
+  "js_wait": 1,
+  "stop_urls": ["https://docs.minaprotocol.com/node-developers"],
+  "selectors_exclude": ["footer"],
   "selectors": {
     "lvl0": {
-      "selector": "(//ul[contains(@class,'menu__list')]//a[contains(@class, 'menu__link menu__link--sublist menu__link--active')]/text() | //nav[contains(@class, 'navbar')]//a[contains(@class, 'navbar__link--active')]/text())[last()]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Documentation"
+      "selector": "",
```

Comment:
Just curious, is it necessary to have a selector with an empty value here?

Reply:
Certainly not necessary! This mostly tells algolia that all indexed content is part of the "Documentation" category as specified by the default value. It's something that I saw in an example config so I'm using it here!